website/pages/community/index.jsx

```diff
@@ -16,28 +16,19 @@ export default function CommunityPage() {
       <VerticalTextBlockList
         data={[
           {
-            header: 'IRC',
-            body: '`#packer-tool` on Freenode',
+            header: 'Community Forum',
+            body:
+              '[Packer Community Forum](https://discuss.hashicorp.com/c/packer)',
           },
           {
             header: 'Announcement List',
             body:
               '[HashiCorp Announcement Google Group](https://groups.google.com/group/hashicorp-announce)',
           },
-          {
-            header: 'Discussion List',
```

Comment:
The Packer Google Group is a pretty active forum with some long time Packer community members helping out with Packer related questions. It is especially helpful when it comes to finding historical content around issues encountered by new Packer users. It would be great if we can encourage the use of the community forum but keep a  link to the Google group available, possibly with some text indicating that it is available for historical context, as an alternative resource. What do you think?

Reply:
@nywilken I like this idea, thank you for raising it!
We will be sunsetting the interactive functionality of the Google Groups over the next few months (exact timelines to be assessed tool-by-tool), and making them announce-only as part of this overall initiative. Happy to leave the Group on the Community page underneath link out to Forum, if you agree. When we do determine timeline to make the Google Group announce-only, I can submit another PR to add a note that the group is reference to historical context. Let me know what you think!

Reply:
@alexandrafreeman that sounds good to me. Thanks for the context on the group becoming an announce-only in the future.